src/QtLocationPlugin/QGCMapUrlEngine.cpp

```diff
@@ -75,6 +75,12 @@ UrlFactory::UrlFactory() : _timeout(5 * 1000) {
     _providersTable["VWorld Satellite Map"] = new VWorldSatMapProvider(this);
 
     _providersTable["Airmap Elevation"] = new AirmapElevationProvider(this);
+
+    _providersTable["Japan Std"] = new JapanStdMapProvider(this);
```

Comment:
Is there some more specific naming for these maps than "Japan foo"? That is way too generic.

Reply:
I change std to contour.

Reply:
I"m not talking about "Std" versus contour. I'm talking about only identifying this class of maps with the prefix "Japan". That provides no detail as to what service is providing these maps. How is a user supposed to understand where these maps come from.

Reply:
Which leads me to also think there should be some other piece of more detailed information for each map provider that is user visible. So that these more specific custom area maps can tell the user some more information than just a name. But for now all we have is a name, so that is the only place we have to provide at least some level of detail.

Reply:
@DonLakeFlyer san. I get the maps from the Geospatial Information Authority of Japan
I think the official name is as follows.
I think this is a long string of characters.
What should I do?

Name:.
Geospatial Information Authority of Japan

Reply:
It looks like from here: https://www.gsi.go.jp/ENGLISH/ that they use the initials GSI to refer to themselves. So how about "Japan (GSI)" as a prefix.

Reply:
@DonLakeFlyer san. Thank you. I agree. I'll change.

Reply:
I want to register more than one GSI map.
When I changed it to "Japan(GSI)", the map type was not displayed.
I think "Japan-GSI" is fine.

![Screenshot from 2020-09-03 06-57-53](https://user-images.githubusercontent.com/646194/92041687-ab793e00-edb3-11ea-9fe1-b821dd9c6c35.png)

Reply:
Yes, that's fine